packages/plugin-infra/src/PostContext.ts

```diff
@@ -60,6 +69,7 @@ export interface PostContext extends PostContextAuthor {
     /** Auto computed */
     readonly identifier: Subscription<null | PostIdentifier>
     readonly url: Subscription<URL | null>
+    readonly coAuthors: Subscription<PostContextCoAuthor[] | null>
```

Comment:
```suggestion
    readonly coAuthors: Subscription<PostIdentifier[] | null>
```

Reply:
The coAuthors need more information, like `name`.

Reply:
I mean `ProfileIdentifier`. So I found we have `PostContextAuthor`. Can we just keep using it?